components/Code/index.tsx

```diff
@@ -69,7 +69,11 @@ const Code = (props: CodeProps) => {
   };
 
   const code = value ?? (Array.isArray(children) ? children[0] : children) ?? '';
-  const highlightedCode = syntaxHighlighter && code ? syntaxHighlighter(code, language, codeOpts, { mdx: true }) : code;
+
+  const highlightedCode =
+    syntaxHighlighter && typeof syntaxHighlighter === 'function' && code
```

Comment:
I ran into an error here when rendering plain markdown with a `<code>` tag in it in [this unit test](https://github.com/readmeio/markdown/pull/1116/files#diff-76dd675e9cd6b79e2d83c5e4286d136368eb9ad49230fe409c62758bb6111ff5R82) so adding an additional check on `typeof SyntaxHighlighter`